.github/workflows/manual-release.yml

```diff
@@ -0,0 +1,40 @@
+name: Manual - Produce Docker Image, GH Release
+
+on:
+  push:
+    branch:
+      - controller-0.51.0
+
+jobs:
+  # Manual workflow to backfill any failed/missed builds
+  # Update the variable TAG below as well as line #5 in values-sigsci.yaml file
+  # This action will build and push the tagged in imaged to docker hub, build and push Github Tag
+  build_push_release:
+    runs-on: ubuntu-latest
+    env:
+      TAG: "0.51.0"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_API_TOKEN }}
+      - name: Build and push Docker image
+        env:
+          REPO: signalsciences/sigsci-nginx-ingress-controller
+        run: |
+          docker build . --file Dockerfile --build-arg NGINX_INGRESS_VERSION=v${TAG} --tag ${REPO}:${TAG}
+          docker push ${REPO}:${TAG}
+
+      - name: Create Github Release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ env.TAG }}
+          release_name: Match release ${{ env.TAG }}
+          body: |
+            Match release kubernetes/ingress-nginx v${{ env.TAG }}
+          draft: false
+          prerelease: false
```

Comment:
This manual release action probably should not be in this branch or in main; if we add a manual release action, it should take inputs at the time it is triggered so you don't need to update the action for every version you want to manually release.